appveyor.yml

```diff
@@ -15,6 +15,7 @@ install:
   #- ps: (new-object net.webclient).DownloadFile('https://raw.github.com/pypa/pip/master/contrib/get-pip.py', 'C:/get-pip.py')
   #- "%PYTHON%/python.exe C:/get-pip.py"
   #- "%PYTHON%/Scripts/pip.exe install --upgrade setuptools"
+  - "%PYTHON%/Scripts/pip.exe install pytest --no-use-wheel"
```

Comment:
Please a comment explaining why whells were disable with a link to the py.test and pip issues. 
You should also add a note in the docs (I guess the right place is the README file that contains information for developers).